packages/employer-client/src/Claims/ClaimCreate.tsx

```diff
@@ -1,11 +1,21 @@
+import React from "react"
```

Comment:
May be preferable to import useEffect and useState individually, to avoid unnecessary imports included in React, and for consistency with other places in this repo.

Reply:
This has now been changed to import only the hooks from React